src/app/components/service-down/service-down.component.html

```diff
@@ -1,8 +1,10 @@
 <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-xl">Sorry, there is a problem with the service</h1>
-      <p class="govuk-body">Try again later.</p>
+      <h1 class="govuk-heading-xl">{{(currentError)?.header}}</h1>
```

Comment:
this might introduce empty h1, not sure if you want an ngIf etc?

Reply:
I removed the nullable now. It always should have a error message [it has a default value] I have reused the existing component. if there is no error message default error will be shown. Thanks